config/default.json

```diff
@@ -39,7 +39,17 @@
 
   //Security-related parameters.
   "security": {
-    "secret": "such secret"
+    "secret": "such secret",
```

Comment:
Should not include  default secret. I want applications to fail in production is no secret has been made.